pkg/test/e2e/e2eutil/spec_util.go

```diff
@@ -76,3 +77,26 @@ func NewTestWriteReadJob(namespace string, controllerUri string) *batchv1.Job {
 		controllerUri, controllerUri)
 	return newTestJob(namespace, command)
 }
+
+func GetTier2(namespace string) *corev1.PersistentVolumeClaim {
```

Comment:
The function name seems wrong to me. `NewTier2` probably sounds better and is consistent with the rest of the funcs.

Reply:
Yeah... I was trying to make it consistent, but this yaml is the Go version of /test/e2e/resource/tier2.yaml. I am not sure if New will make it confusing because what the test code does it to get that deployed tier2 and delete it. If you think New is better I am cool with it.

Reply:
I understand your concern, but since it's creating a new spec, `NewTier2` seems a better option.

Reply:
Done